Logisim-Fork/src/main/java/com/cburch/logisim/gui/start/Startup.java

```diff
@@ -401,8 +401,14 @@ public boolean autoUpdate(boolean FromMain, Frame frame) {
 					"Although an Internet connection should be available, the system couldn't connect to the URL requested by the auto-updater\nIf the error persist, please contact the software maintainer\n-- AUTO-UPDATE ABORTED --");
 			return (false);
 		}
-		ArgonXML logisimData = new ArgonXML(in, "logisim");
-
+		ArgonXML logisimData;
+		try {
+			logisimData = new ArgonXML(in, "resources/logisim");
```

Comment:
This

Reply:
> 
```suggestion
			logisimData = new ArgonXML(in, "logisim");
```

Reply:
That's seems more than reasonable, actually i think it was a mistaken from the IDE _(another reason to not use them)_,
if i recall correctly i added than try because the build would crash and i didn't figured out why

So I am going to just revert the "Optional Update check" commmit

Reply:
Sorry for force-push but after some thinking it seemed the best solution without creating a new PR